packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -248,6 +249,38 @@ class InteractionResponses {
     this.replied = true;
   }
 
+  /**
+   * An object containing the same properties as CollectorOptions, but a few more:
```

Comment:
This isn't more, it's [less](https://discord.js.org/#/docs/discord.js/main/typedef/CollectorOptions)!